cmd/hypper/install.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/cmd/helm/require"
+	helmAction "helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	helmCli "helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/downloader"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+var installDesc = `install a helm chart by wrapping helm calls (for now)`
+
+func newInstallCmd(actionConfig *helmAction.Configuration, out io.Writer) *cobra.Command {
+	client := helmAction.NewInstall(actionConfig)
+	valuesOpts := &values.Options{}
+	cmd := &cobra.Command{
+		Use:   "install [NAME] [CHART]",
+		Short: "install a chart",
+		Long:  installDesc,
+		Args:  require.MinimumNArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			// TODO use output package for formatting:
+			_, _ = out.Write([]byte(esPrintf(":cruise_ship: Installing %s…", args[0])))
+			// TODO decide how to use returned rel:
+			_, err := runInstall(args, client, valuesOpts, out)
+			if err != nil {
+				return err
+			}
+			_, _ = out.Write([]byte(esPrint(":goal: Done!\n")))
+			return nil
+		},
+	}
+	return cmd
+}
+
+func runInstall(args []string, client *helmAction.Install, valueOpts *values.Options, out io.Writer) (*release.Release, error) {
+	var helmSettings = helmCli.New()
+	fmt.Println(client.Version)
+
+	// TODO add hypper specific code here
+
+	if client.Version == "" && client.Devel {
```

Comment:
Sorry I dont understand this. Is this part of helm upstream or are we using it to skip over charts not having versions?

Reply:
This comes from Helm's https://github.com/helm/helm/blob/master/cmd/helm/install.go#L174-L177.
Seems to be a catch-all in case that a chart version was not specified up until here.

Reply:
mmmh, I was looking at the wrong part `pkg/action` :facepalm: 

Thanks!